test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -528,13 +517,11 @@ def wait_and_remove(self, filename):
         """
         for _ in range(10):
             if not os.path.isfile(filename):
-                time.sleep(1)
-                continue
+                break
             try:
                 os.remove(filename)
             except:
-                pass
-            break
+                time.sleep(1)
```

Comment:
Do you want to complain if you did this 10 times and the file is still not deleted?

Reply:
IMHO, no sense in complaining if there's nothing that can be done.

Reply:
Will your test fail?  If so, it might be nice to have some indication of why the test failed.